misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/typemodel/BallerinaOpenApiType.java

```diff
@@ -37,6 +38,24 @@
     private List<Tag> tags;
     private List<BallerinaOpenApiPath> pathList;
     private BallerinaOpenApiComponent component;
+    private List<Tag> filteredTags;
+    private List<Operation> operations;
+
+    public List<Operation> getOperations() {
+        return operations;
+    }
+
+    public void setOperations(List<Operation> operations) {
+        this.operations = operations;
+    }
+
+    public List<Tag> getFilteredTags() {
```

Comment:
This is not used anywhere. Shall we remove this?

Reply:
`operatoins` attribute I used for the future purpose of implementation. Need to have some clarification with team its further implementation.